src/helpers/pdf-helpers.js

```diff
@@ -63,14 +64,12 @@ const generatePDF = (content, outputDirectory) => {
     });
 };
 
-const generatePDFFromMarkdown = (content, outputDirectory, parentDirectory = '') => {
-  content.forEach(item => {
-    if (fileShouldBeIgnored(item)) {
-      return;
-    }
+const generatePDFFromMarkdown = (content, mdDocs) => {
+  // Order by type to ensure that we get the folder content first and the subfolder after
+  _.orderBy(content.filter(x => !fileShouldBeIgnored(x)), 'type', 'desc').forEach(item => {
```

Comment:
The issue was that the file of the subchapters were added first to the processor and then the chapter content itself. Thus we had 3.1 3.2.3.3 and then the content of 3